src/utils/farmsPriceHelpers.ts

```diff
@@ -0,0 +1,12 @@
+import { Farm } from 'state/types'
+
+export const filterFarmsByQuoteToken = (farms: Farm[], preferredQuoteTokens: string[] = ['BUSD', 'wBNB']): Farm => {
+  const preferredFarm = farms.find((farm) => {
+    return preferredQuoteTokens.some((quoteToken) => {
+      return farm.quoteToken.symbol === quoteToken
+    })
+  })
+  return preferredFarm || farms[0]
```

Comment:
Farm filtering logic, to return the first BUSD or BNB farm if possible.

Reply:
Comment please.